kpi/utils/log.py

```diff
@@ -1,10 +1,13 @@
 # -*- coding: utf-8 -*-
 import logging as stdlib_logging
 
+
 def get_default_logger():
     return stdlib_logging.getLogger("console_logger")
 
+
 # logging does not print to stdout when RAVEN/sentry is activated.
 # To avoid to call logging.getLogger everywhere, we overload logging
 logging = get_default_logger()
 
+
```

Comment:
PEP-8 really requires `\n\n\n` at the end of a file? I guess I don't like the BDFL's taste in whitespace :astonished:
(just commentary, not a request for any change to the code)

Reply:
No, I made the mistake. 
There are too many. There should be only 2.